modules/services/sharing/src/main/kotlin/au/com/shiftyjelly/pocketcasts/sharing/SharingLogger.kt

```diff
@@ -1,18 +1,28 @@
 package au.com.shiftyjelly.pocketcasts.sharing
 
+import au.com.shiftyjelly.pocketcasts.analytics.AnalyticsEvent
 import au.com.shiftyjelly.pocketcasts.utils.log.LogBuffer
 import timber.log.Timber
 
 internal class SharingLogger : SharingClient.Listener {
-    override fun onShare(request: SharingRequest) {
-        val message = "Sharing: $request"
+    override fun onShare(
+        request: SharingRequest,
+        event: AnalyticsEvent,
+        eventProperties: Map<String, Any>,
+    ) {
+        val message = "Sharing: $request $event $eventProperties"
```

Comment:
Is it needed? This will log analytics events to the `LogBuffer` which wouldn't be helpful for debugging. And we already log all analytics events in `TracksAnalyticsTracker`.